posthog/api/insight.py

```diff
@@ -80,6 +80,7 @@ class Meta:
             "order",
             "deleted",
             "dashboard",
+            "dive_dashboard",
```

Comment:
@paolodamico do I need to add it to the `InsightBasicSerializer` too (it seems similar to color, but I'm not sure)?